unit-10/main.js

```diff
@@ -0,0 +1,60 @@
+// exercise 1
+function getMultiplicationTable(number) {
```

Comment:
```suggestion
function getMultiplicationTable (number) {
```
Khi khai báo 1 hàm thì thêm dấu cách trước dấu `(`